src/components/events/EventsSection.jsx

```diff
@@ -18,23 +19,29 @@ class EventsSection extends React.Component {
   }
 
   componentDidMount() {
-    db.collection('events').onSnapshot((querySnapshot) => {
-      const events = [];
-      querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, eventDate } = doc.data();
+    const events=[];
+    axios.get('http://localhost:5000/events').then((response) => {
+      for (var key in response.data){
+        const e = response.data[key]
```

Comment:
instead of accessing each of these, lets just inline them so it's easier to read

Reply:
Not sure exactly what is meant by inline... are they considered inline here?
https://github.com/uwblueprint/dancefest-web/pull/94/commits/14df67de75375d7d3c265cac721d763d08a9eb69